### ISSUE TITLE
Add Github accessibility alt-text bot bot

### ISSUE THREAD

**Issue by aleesteele** [alan-turing-institute/the-turing-way#3207]
### Summary

On Slack, @da5nsy kindly shared that Github released a new accessibility-alt-text-bot, which will leave automated reminders when people share images without meaningful alt text: https://github.blog/2023-06-12-make-your-github-projects-more-accessible-with-accessibility-alt-text-bot/
